apps/server/src/modules/account/uc/account.uc.spec.ts

```diff
@@ -913,6 +936,82 @@ describe('AccountUc', () => {
 				accountUc.findAccountById({ userId: mockSuperheroUser.id } as ICurrentUser, { id: 'xxx' } as AccountByIdParams)
 			).rejects.toThrow(EntityNotFoundError);
 		});
+		it('should throw, if target account has no user', async () => {
+			await expect(
+				accountUc.findAccountById({ userId: mockSuperheroUser.id } as ICurrentUser, { id: 'xxx' } as AccountByIdParams)
+			).rejects.toThrow(EntityNotFoundError);
+		});
+	});
+
+	describe('createAccount', () => {
+		afterEach(() => {
+			jest.clearAllMocks();
+		});
+
+		it('should sanitize username for local user', async () => {
+			const spy = jest.spyOn(accountService, 'create');
+			const params: AccountCreateParams = {
+				username: ' John.Doe@domain.tld ',
+			};
+			await accountUc.createAccount(params);
+			expect(spy).toHaveBeenCalledWith({
+				username: 'john.doe@domain.tld',
+			});
+		});
+		it('should not sanitize username for external user', async () => {
+			const spy = jest.spyOn(accountService, 'create');
+			const params: AccountCreateParams = {
+				username: ' John.Doe@domain.tld ',
+				systemId: 'ABC123',
+			};
+			await accountUc.createAccount(params);
+			expect(spy).toHaveBeenCalledWith(
+				expect.objectContaining({
+					username: ' John.Doe@domain.tld ',
```

Comment:
Would be nicer to check for the sanitized user name as in the previous test

Reply:
But the test should test, that the username will not be sanitized.

Reply:
Sorry, my mistake